src/Sql/PostgreSQL/Functions/user_update.sql

```diff
@@ -0,0 +1,75 @@
+drop function user_update(UUID,VARCHAR,VARCHAR,INT4,VARCHAR,VARCHAR,VARCHAR,VARCHAR,TEXT,VARCHAR,TEXT,TEXT,TIMESTAMPTZ,TEXT,TEXT,TEXT,INT4,TEXT,TEXT,INT8,INT2,INT2,VARCHAR,VARCHAR,VARCHAR,INT4,INT4,TIMESTAMPTZ,TIMESTAMPTZ);
+
+CREATE OR REPLACE FUNCTION user_update
+(
+    _id                                 UUID,
+    _name                               VARCHAR,
+    _email                              VARCHAR,
+    _email_verified                     INT,
+    _master_password                    VARCHAR,
+    _master_password_hint               VARCHAR,
+    _culture                            VARCHAR,
+    _security_stamp                     VARCHAR,
+    _two_factor_providers               TEXT,
+    _two_factor_recovery_code           VARCHAR,
+    _equivalent_domains                 TEXT,
+    _excluded_global_equivalent_domains TEXT,
+    _account_revision_date              TIMESTAMPTZ,
+    _key                                TEXT,
+    _public_key                         TEXT,
+    _private_key                        TEXT ,
+    _premium                            INT,
```

Comment:
Why is this passed as int and cast to bit later?

Reply:
actually ive fixed that timestamp issue now, i was messing with datatypes and missed that one initially when i was passing through NULLS and having issues due to the case (ie null vs NULL)

im recasting some of the datatypes within the functions (ie int/smallint) due to the values being passed as INT and having issues.

also bit v boolean, its better/easier to have the datatype as boolean to begin with, else it has to be passed in as int, and then cast to bit in the function. 

this is just a weirdness of the function when its called normally, im not sure how your C code process would call it, but with these datatypes, it has to be cast before calling the function, or the function has to do it when doing the insert/update

eg 
select * from fn_some_function(1::BIT,123::SMALLINT) ;
vs
select * from fn_some_function(false,123) ;
vs


for now i've chosen boolean instead of bit, but it can be changed back if this is too much of a change to work with.